contracts/libraries/DnGmxJuniorVaultManager.sol

```diff
@@ -322,13 +323,17 @@ library DnGmxJuniorVaultManager {
         int256 borrowVal = borrowValue.toInt256();
 
         if (borrowVal > state.dnUsdcDeposited) {
+            uint256 diff = uint256(borrowVal - state.dnUsdcDeposited);
+            if (diff < state.rebalanceProfitUsdcAmountThreshold && !_isValidRebalanceHF(state)) return;
             // If glp goes up - there is profit on GMX and loss on AAVE
             // So convert some glp to usdc and deposit to AAVE
-            state.dnUsdcDeposited += _convertAssetToAUsdc(state, uint256(borrowVal - state.dnUsdcDeposited)).toInt256();
+            state.dnUsdcDeposited += _convertAssetToAUsdc(state, diff).toInt256();
         } else if (borrowVal < state.dnUsdcDeposited) {
+            uint256 diff = uint256(state.dnUsdcDeposited - borrowVal);
+            if (diff < state.rebalanceProfitUsdcAmountThreshold) return;
```

Comment:
in the previous, if  condition, max loss on aave is bound by HF condition check, should there be something similar to limit gmx loss?